build_deploy.sh

```diff
@@ -4,7 +4,7 @@ TAG=$(git rev-parse --short=7 HEAD)
 
 BM_INVENTORY_IMAGE="quay.io/app-sre/bm-inventory"
 
-docker build -t "${BM_INVENTORY_IMAGE}:latest" -f Dockerfile.demo .
+docker build -t "${BM_INVENTORY_IMAGE}:latest" -f Dockerfile.bm-inventory .
```

Comment:
Why not Dockerfile?
Why do you have so many docker files?

Reply:
* `Dockerfile.s3-object-expirer` was removed in `bm-inventory` commit: `cc09b177af6643bbf6c2691fe24b8a093a55c20b`
* `Dockerfile.XXX-build` is used internally for development environment by skipper (this is how `bm-inventory` has decided to implement it)
* `Dockerfile.assisted-installer` is used for deployment as the production image
* `Dockerfile.test` was I created by @apahim so we can run make check on PRs. The CI will execute the pr_check.sh , but since we can't install dependencies in the CI, we build and run a docker container using the Dockerfile.test.
* `Dockerfile.demo` was created by @apahim in order to create the full chain of deployment instead of `Dockerfile.bm-inventory` and it can be removed once this PR is merged.